app/reflexes/send_invoice_reflex.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SendInvoiceReflex < ApplicationReflex
+  def confirm(order_ids)
+    Spree::Order.where(id: order_ids).find_each do |o|
+      Spree::OrderMailer.invoice_email(o.id).deliver_later unless o.distributor.can_invoice?
```

Comment:
Send an invoice if we can't send an invoice? :exploding_head:

Reply:
oops ;) Thanks ! f45819c26f